datahub-web-react/src/conf/theme/theme_light.config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "styles": {
+        "@border-radius-base": "8px",
```

Comment:
Why do these need the @ thing?

Reply:
We don't need it here, but this is the format of less style variables that antd uses:

https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less

When we merge them with the antd less variables we need the `@` signs. Whether we add theme here or add them in craco.config.js (https://github.com/linkedin/datahub/pull/2193/files#diff-fabf82065057e31dc626906f0279bc0ca3b309c3d964a9c4d5b5ff25f5beb5b0R11), either would be the same